### PR TITLE
chore: bump playwright and go

### PR DESCRIPTION
**Pull request by DmitriyMV**
Bump both at the same time.

### CODE REVIEW

File: .kres.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      runners:
+        - self-hosted
+        - pkgs
+      steps:
+        - name: build
+          nonMakeStep: true
+          command: make
+        - name: login-to-registry
+          conditions:
+            - only-on-tag
```
Comment (smira) [inline .kres.yaml:22]: do you want to push only on tag? or you should just push on every `main` branch merge?

(this is a copy-paste from kubelet, which is special)

**Review by smira — APPROVED** [review]

File: .kres.yaml (hunk 1 above)
Comment (DmitriyMV) [reply .kres.yaml]: Removed. Thanks, I think there is no need for push on tag here.

Comment (smira) [reply .kres.yaml]: Removed was a wrong move I guess. You should have had a condition to push only on merges to `main`?

Comment (DmitriyMV) [reply .kres.yaml]: Yeah, I also forgot to run rekres. Fixed here: https://github.com/siderolabs/playwright-go/pull/7

**Pull request merged**